src/Go/boardAnalysis/goAI.ts

```diff
@@ -403,7 +403,9 @@ async function getIlluminatiPriorityMove(moves: MoveOptions, rng: number): Promi
     return moves.corner()?.point ?? null;
   }
 
-  const hasMoves = [moves.eyeMove(), moves.eyeBlock(), moves.growth(), moves.defend, surround].filter((m) => m).length;
+  const hasMoves = [moves.eyeMove(), moves.eyeBlock(), moves.growth(), moves.defend(), surround].filter(
```

Comment:
```suggestion
  const hasMoves = [moves.growth(), moves.defend(), surround].some(Boolean);
```
`some` is the correct function to use here, and `Boolean` is the right function to check for truthiness. `eyeMove` and `eyeBlock` must be false, since if they were true we would have already returned.